sdk/python/kubeflow/training/constants/constants.py

```diff
@@ -83,18 +101,15 @@
 
 # PyTorchJob constants
 PYTORCHJOB_KIND = "PyTorchJob"
+PYTORCHJOB_MODEL = "KubeflowOrgV1PyTorchJob"
```

Comment:
Any reason to override?

Reply:
@johnugeorge What do you mean by override here ?
I just make string type rather than object to reduce number of typing errors in Pylance.